app/experimenter/templates/experiments/radio_widget_closer.html

```diff
@@ -0,0 +1,7 @@
+{% for group, options, index in widget.optgroups %}
+  {% for option in options %}
+		<div class="radio ml-4">
+			{% include option.template_name with widget=option %}
+		</div>
+  {% endfor %}
+{% endfor %}
```

Comment:
Here is the custom widget that I referenced earlier in this review. I use `ml` to accomplish the right spacing.